CalibPPS/AlignmentGlobal/plugins/PPSAlignmentHarvester.cc

```diff
@@ -232,18 +236,19 @@ void PPSAlignmentHarvester::dqmEndRun(DQMStore::IBooker& iBooker,
   });
 
   bool doXAli = false, doXAliRel = false, doYAli = false;
-  for (unsigned int i = 0; i < sequence_.size(); i++) {
-    if (sequence_[i] == "x_alignment") {
-      xAlignment(iBooker, iGetter, cfg, cfg_ref, i);
+  for (const std::string& aliMethod : sequence_) {
+    if (aliMethod == "x_alignment") {
+      xAlignment(iBooker, iGetter, cfg, cfg_ref);
       doXAli = true;
-    } else if (sequence_[i] == "x_alignment_relative") {
-      xAlignmentRelative(iBooker, iGetter, cfg, i);
+    } else if (aliMethod == "x_alignment_relative") {
+      xAlignmentRelative(iBooker, iGetter, cfg);
       doXAliRel = true;
-    } else if (sequence_[i] == "y_alignment") {
-      yAlignment(iBooker, iGetter, cfg, i);
+    } else if (aliMethod == "y_alignment") {
+      yAlignment(iBooker, iGetter, cfg);
       doYAli = true;
     } else
-      edm::LogError("PPS") << "[harvester] " << sequence_[i] << " is a wrong method name.";
+      edm::LogError("PPS") << "[harvester] " << aliMethod << " is a wrong method name.";

```

Comment:
```suggestion
      edm::LogError("PPSAlignmentHarvester") << aliMethod << " is a wrong method name.";
```
In the parentheses of the LogError you need to put the class name

Reply:
Ok, I'll change the log categories to the proper classes. I remember that once we've decided to use "PPS" everywhere, but now I don't remember why ;)